### ISSUE TITLE
Table.from_csv fails in ipython

### ISSUE THREAD

**Issue by mikejcorey** [onyxfish/agate#335]
Super weird. This works fine in python, but fails in ipython:

import csv
import agate

text_type = agate.Text()
number_type = agate.Number()

columns = (
    ('Address', text_type),
    ('Description', text_type),
    ('Plant Type', text_type),
    ('Plant Factor', number_type),
    ('Area (m2)', number_type),
)

estimates = agate.Table.from_csv('my/csv/path.csv', columns)

/home/michaelcorey/.Envs/wetprince/lib/python2.7/site-packages/agate/table.pyc in from_csv(cls, path, column_info, row_names, header, **kwargs)
    208         else:
    209             with open(path) as f:
--> 210                 rows = list(csv.reader(f, **kwargs))
    211 
    212         if header:

AttributeError: 'module' object has no attribute 'reader'

Note: Fails with or without type inference.

**Comment by onyxfish** [comment]
Huh, that's bizarre. I'll take a closer look.

**Comment by onyxfish** [comment]
So, I can't replicate this in my 3.5 environment, but I did create a similar problem entirely by accident when I tried to use ipython in my Python2.7 environment and for some reason ipython was still trying to read the Py3 dependencies. I've honestly never seen this before, but I doubt it's a coincidence that happened to me. Are you by chance using pyenv? (I just started so I'm still getting my head around how it changes dependency references.)
